server/user_account/serializers.py

```diff
@@ -43,4 +43,4 @@ class Meta:
 class ClientGridSerializer(serializers.ModelSerializer):
     class Meta:
         model = CustomUser
-        fields = ['first_name', 'last_name', 'role', 'is_active']
+        fields = ['first_name', 'last_name', 'role', 'is_active', 'id', 'email']
```

Comment:
So this will be modified because we don't want id and email to be returned for all the users in the table, We just want this extra info when we 'get' a specific user. But I know that this is temporary, and will be done through the UserSerializer.

Reply:
yep exactly, and I think this entire class was used exactly for that, so we're probably going to have to refactor and remove this whole class anyway